frappe/public/js/frappe/form/quick_entry.js

```diff
@@ -57,6 +58,8 @@ frappe.ui.form.QuickEntryForm = class QuickEntryForm {
 		this.mandatory = fields.filter(df => {
 			return ((df.reqd || df.bold || df.allow_in_quick_entry) && !df.read_only && !df.is_virtual);
 		});
+
+		this.show_edit_in_full_page_button = !(this.mandatory === fields);
```

Comment:
Let's just leave this out? We should let people go to full view regardless of this. 

There can be read-only fields that are computed from the mandatory fields that sometimes help (like _fetch from_s)